app/page.tsx

```diff
@@ -488,21 +491,42 @@ export default function Home() {
             sx={{
               paddingTop: "20px",
               paddingBottom: "10px",
+              '& .break': {
+                display: 'none',
+              },
+              '@media (min-width: 700px)': {
+                '& .break': {
+                  display: 'inline',
+                },
+              },
             }}
           >
-            No trusted hardware. No trusted attestation servers. Only trust zero
-            knowledge proofs, smart contracts, email, and DNS infrastructure.
-            All MIT open source libraries.
+            No trusted hardware. No trusted attestation servers. <span className="break"><br /></span>
```

Comment:
Hey, don't use break like this. If you want to adjust the content in one line, play with the width of the container div